config.go

```diff
@@ -30,12 +31,18 @@ import (
 	"github.com/hazelcast/hazelcast-go-client/types"
 )
 
+const (
+	defaultMaxToleratedMissCount            = 10
```

Comment:
we can add the location of these constants in Java client as a comment here

`com.hazelcast.internal.nearcache.impl.invalidation.RepairingTask`

Reply:
https://github.com/hazelcast/hazelcast-go-client/pull/807/commits/ebcf2b3b7fe69e1af93821996e0b1ea1d524c69d